docs/getting-started/README.md

```diff
@@ -4,39 +4,46 @@ order: 1
 
 # Getting Started
 
+This document will guide you through setting up Kestra in a few minutes.
 
-## Before you begin
-Make sure the followings are already installed:
-- [Docker](https://docs.docker.com/engine/install/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+## Prerequisites
 
-## Launch Kestra
+Make sure the following is installed:
+- [Docker Desktop](https://docs.docker.com/get-docker/)
 
-- Download the Docker Compose file [here](https://github.com/kestra-io/kestra/blob/develop/docker-compose.yml) and save it with the name `docker-compose.yml`. For linux and MacOS, you can run `wget https://raw.githubusercontent.com/kestra-io/kestra/develop/docker-compose.yml`.
-- Run `docker-compose up -d`.
-- Open [http://localhost:8080](http://localhost:8080) in your browser.
+## Run Kestra
 
+### Download Docker Compose file
 
-::: warning
-The default amount of memory available for Docker on MacOS is often not enough to get Kestra up and running with all dependencies. If enough memory is not allocated, it might lead to the Kestra instance continuously restarting. You should allocate at least 4GB memory for the Docker Engine (ideally 8GB). You can check and change the amount of memory in [Resources](https://docs.docker.com/docker-for-mac/#resources).
+Download the Docker Compose file for Kestra using the following command: 
 
-You can also check if you have enough memory by running this command:
 ```bash
-docker run --rm "debian:buster-slim" bash -c 'numfmt --to iec $(echo $(($(getconf _PHYS_PAGES) * $(getconf PAGE_SIZE))))'
+curl -o docker-compose.yml https://raw.githubusercontent.com/kestra-io/kestra/develop/docker-compose.yml
 ```
-:::
 
+If you don't have curl installed, you can download the [Docker Compose file](https://github.com/kestra-io/kestra/blob/develop/docker-compose.yml) and save it as `docker-compose.yml`. 
 
-## Create your first flow
-The default installation contains some examples you can access on the `Flows` menu item.
+### Start your server
 
-We will create a new one:
+Make sure the Docker Desktop app is running. Then, run the following command:
 
-![Flow Create](./assets/flow-2.png)
+```bash
+docker-compose up -d
```

Comment:
docker compose command is the new docker-compose.

Reply:
I didn't know that. Thanks :)